test/android/core/javatests/src/org/xwalk/core/xwview/test/ShouldOverrideUrlLoadingTest.java

```diff
@@ -417,7 +417,7 @@ public Boolean call() throws Exception {
         // Since the targetURL was loaded from the test server it means all processing related
         // to dispatching a shouldOverrideUrlLoading callback had finished and checking the call
         // is stable.
-        assertEquals(shouldOverrideUrlLoadingCallCount + 1,
+        assertEquals(shouldOverrideUrlLoadingCallCount,
```

Comment:
yes, but do you think it's better to use shouldOverrideUrlLoadingCallCount which is get from mShouldOverrideUrlLoadingHelper.getCallCount() before getting request from web server instead of using "0" directly?